RealmModelGenerator/Entity.swift

```diff
@@ -90,7 +90,7 @@ class Entity {
         
         while relationships.contains({$0.name == name}) {
             count++
-            name = "Relationship\(count)"
+            name = "\(name)\(count)"
```

Comment:
with this logic, each time you rebuild name it will be Relationship1 then Relationship12 then Relationship123